### PR TITLE
Fix css debt

### PR DESCRIPTION
**Pull request by DilwoarH**
https://trello.com/c/Sr6cZpfb/72-code-tidy-up-from-monday-night-rapid-deploys

### CODE REVIEW

File: app/views/coronavirus_landing_page/_page_header.html.erb [hunk 1]
```diff
@@ -43,10 +46,10 @@
...
       <div class="govuk-grid-column-two-thirds">
-        <p class="govuk-body govuk-font-colour-white govuk-!-margin-bottom-1">
+        <p class="govuk-body covid__page-header-body-pretext">
           <%= t("coronavirus_landing_page.guidance.pretext") %>
         </p>
-        <p class="govuk-body govuk-font-colour-white">
+        <p class="govuk-body">
           <a class="govuk-link covid__page-header-link" href="<%= t("coronavirus_landing_page.guidance.link.href") %>">
             <%= t("coronavirus_landing_page.guidance.link.text") %>
           </p>
```
Comment (andysellick) [inline app/views/coronavirus_landing_page/_page_header.html.erb]: I think this trailing `</p>` is unnecessary - should be an `</a>`?

File: app/views/coronavirus_landing_page/_page_header.html.erb [hunk 2]
```diff
@@ -43,10 +46,10 @@
 
     <div class="govuk-grid-row covid__page-header-guidance-section">
```
Comment (andysellick) [inline app/views/coronavirus_landing_page/_page_header.html.erb]: I think you could drop the `govuk-grid-row` from this element and fold it into the two thirds column that precedes it?

File: app/views/coronavirus_landing_page/_page_header.html.erb (hunk 1 above)
Comment (sihugh) [reply app/views/coronavirus_landing_page/_page_header.html.erb]: This could be why the tests are all failing - just checking in a branch

Comment (sihugh) [reply app/views/coronavirus_landing_page/_page_header.html.erb]: It was - that fix is merged if you rebase now

File: app/views/coronavirus_landing_page/_page_header.html.erb (hunk 2 above)
Comment (DilwoarH) [reply app/views/coronavirus_landing_page/_page_header.html.erb]: done

File: app/views/coronavirus_landing_page/_page_header.html.erb (hunk 1 above)
Comment (DilwoarH) [reply app/views/coronavirus_landing_page/_page_header.html.erb]: omg! #fail

**Pull request merged**